packages/blockchain-utils-linking/src/cosmos.ts

```diff
@@ -0,0 +1,84 @@
+import { AccountID } from 'caip';
+import { AuthProvider } from './auth-provider';
+import { getConsentMessage, LinkProof } from './util';
+import { Tx, SignMeta } from '@tendermint/sig';
+import { hash } from '@stablelib/sha256';
+import * as uint8arrays from 'uint8arrays';
+
+const CHAIN_ID = "cosmos:cosmoshub-3";
```

Comment:
Should this really be hardcoded? Seems like we'd want to support any cosmos blockchain right?